boards/esp32-wrover-kit/esp32-wrover-kit.config

```diff
@@ -0,0 +1,2 @@
+# Sets up configuration for openocd
+CONFIG_MODULE_ESP_JTAG=y
```

Comment:
But it is enabled by default
https://github.com/RIOT-OS/RIOT/blob/c84a40abc4511f2bc5b55f0db8e708432e32d161/boards/esp32-wrover-kit/Makefile.dep#L5-L6

Reply:
Indeed, you are right. We have discussed it in https://github.com/RIOT-OS/RIOT/pull/16911#issuecomment-948841846. It will still flash with `esptool` unless `PROGRAMMER` is set to `openocd`. I have forgotten this already. We can leave it as is.

Reply:
Another question. Why did you model this only setting within a separate file? Wouldn't it be enough to use
```
select MODULE_ESP_JTAG
```
here https://github.com/RIOT-OS/RIOT/blob/6c7ce3d49c6f3b4ad9929147482be3946c2624f2/boards/esp32-wrover-kit/Kconfig#L10-L14

Reply:
No, `select` forces a module to be `y`, this is configurable by the user (AFAIU), so by using a `.config` file we provide a default configuration.

Reply:
OK, that is `.config` corresponds always to `Makefile.default`.

Reply:
Somehow yes. But one can only configure visible symbols this way (i.e., symbols that the user would be able to configure via menuconfig).